src/containers/CStreamingModelContainer/index.tsx

```diff
@@ -12,15 +12,13 @@ interface CStreamingModelContainer {
 const CStreamingModelContainer = ({ label, details }: CStreamingModelContainer) => {
   return (
     <div className="w-[532px] flex flex-col">
-      <CLabel label={label} details={details} className="mb-1" />
+      <CLabel label={label} details={details} toolTipTitle="Straming model" className="mb-1" />
       <div className="flex gap-2">
         <CStreamingModel isSelected={true} model="linear" />
         <CStreamingModel
           model="exponential"
           disabled
-          onClick={() =>
-            toast('error', 'Exponential streams are saved for a later version.')
-          }
+          onClick={() => toast('error', 'Exponential streams are saved for a later version.')}
```

Comment:
create a function above the return statement called `handleExponentialClick` and write the toast there.